docs/reference/userapi.rst

```diff
@@ -70,3 +75,129 @@ that best suits that file-format.
 .. autoclass:: imageio.core.format.Writer
     :inherited-members:
     :members: 
+
+.. _migration_from_v2:
+
+Migrating to ImageIO v3
+^^^^^^^^^^^^^^^^^^^^^^^
+
+The primary novelty of the v3 API compared to the v2 API is that images are now
+treated as ndimages, a generalization of (flat) 2D images to N dimensions. This
+change allows us to design a much simpler API; however, comes with a few
+backwards incompatible changes.
+
+Avoiding API Migration for Legacy Scripts
+-----------------------------------------
+
+If you have an old script where you do not wish to migrate the API, you can
+explicitly import the old v2 API and avoid migration. Note that, while this will
+keep the old (v2) API, it will use on the new (v3) plugins and backends, which
+may contain other backwards incompatible changes. As such, you should prefer a
+full migration to v3, and should avoid using the v2 API in new code. To import
+the v2 API, depending on your script/code, use::
+
+    import imageio... as iio
+    import imageio... as imageio
```

Comment:
This is invalid syntax and looks confusing. Do you mean `import imageio.v2 as iio`?

Reply:
Yes, I agree. This is still a TODO.  https://github.com/imageio/imageio/pull/720 is not yet merged, and its exact layout determines how this should be imported here.

Will fix it once we are clear on our transition policy.

Reply:
I changed it to `import imageio.v2 as iio` for now. If we end up agreeing on something else in #720 I will update again.